nltk/stem/snowball.py

```diff
@@ -288,6 +295,488 @@ def _rv_standard(self, word, vowels):
+                            '\u0643\u0645\u0627', '\u0647\u0645\u0627'
+                            )
+
+    __suffix_noun_step1b = ('\u0646')
+
+    __suffix_noun_step2a = ('\u0627', '\u064a', '\u0648')
+
+    __suffix_noun_step2b = ('\u0627\u062a')
+
+    __suffix_noun_step2c1 = ('\u062a')
+
+    __suffix_noun_step2c2 = ('\u0629')
+
+    __suffix_noun_step3 = ('\u064a')
+
+    __suffix_verb_step1 = ('\u0647', '\u0643',
+                           '\u0646\u064a', '\u0646\u0627', '\u0647\u0627', '\u0647\u0645', '\u0647\u0646', '\u0643\u0645', '\u0643\u0646',
+                           '\u0647\u0645\u0627', '\u0643\u0645\u0627', '\u0643\u0645\u0648'
+                          )
+
+    __suffix_verb_step2a = ( '\u062a', '\u0627', '\u0646' , '\u064a',
+                             '\u0646\u0627', '\u062a\u0627', '\u062a\u0646', '\u0627\u0646', '\u0648\u0646', '\u064a\u0646',
+                             '\u062a\u0645\u0627'
+                           )
+
+    __suffix_verb_step2b = ('\u0648\u0627','\u062a\u0645')
+
+    __suffix_verb_step2c = ('\u0648',
+                            '\u062a\u0645\u0648'
+                           )
+
+    __suffix_all_alef_maqsura = ('\u0649')
+
+    # Prefixes
+    __prefix_step1 = ('\u0623',
+                      '\u0623\u0623', '\u0623\u0622', '\u0623\u0624', '\u0623\u0627', '\u0623\u0625',
+                      )
+
+    __prefix_step2a = ('\u0641\u0627\u0644', '\u0648\u0627\u0644')
+
+    __prefix_step2b = ('\u0641', '\u0648')
+
+    __prefix_step3a_noun = ('\u0627\u0644', '\u0644\u0644',
+                            '\u0643\u0627\u0644', '\u0628\u0627\u0644',
+                            )
+
+    __prefix_step3b_noun = ('\u0628', '\u0643', '\u0644',
+                            '\u0628\u0628', '\u0643\u0643'
+                           )
+
+    __prefix_step3_verb = ('\u0633\u064a', '\u0633\u062a', '\u0633\u0646', '\u0633\u0623')
+
+    __prefix_step4_verb = ('\u064a\u0633\u062a', '\u0646\u0633\u062a', '\u062a\u0633\u062a')
+
+    is_verb = True
+    is_noun = True
+    is_defined = False
+
+    suffixes_verb_step1_success = False
+    suffix_verb_step2a_success = False
+    suffix_verb_step2b_success = False
+    suffix_noun_step2c2_success = False
+    suffix_noun_step1a_success = False
+    suffix_noun_step2a_success = False
+    suffix_noun_step2b_success = False
+    suffixe_noun_step1b_success = False
+    prefix_step2a_success = False
+    prefix_step3a_noun_success = False
+    prefix_step3b_noun_success = False
+
+    def __normalize_pre(self, token):
+        """
+        :param token: string
+        :return: normalized token type string
+        """
+        # strip diacritics
+        token = self.__vocalization.sub('', token)
+        #strip kasheeda
+        token = self.__kasheeda.sub('', token)
+        # strip punctuation marks
+        token = self.__arabic_punctuation_marks.sub('', token)
+
+        return token
+
+    def __normalize_post(self, token):
+
```

Comment:
@LBenzahia would you mind removing blank lines after function definitions please?